grr/core/grr_response_core/lib/casing_test.py

```diff
@@ -0,0 +1,32 @@
+# Lint as: python3
+"""Tests for casing functions."""
+
+from grr_response_core.lib import casing
+
+from grr.test_lib import test_lib
+
+
+class CasingTests(test_lib.GRRBaseTest):
```

Comment:
There should be a test class per class or function. In your case, it should be `SnakeToCamelTest` and `CamelToSnakeTest`.

Reply:
Thank you! I split the tests into the two new test classes.